app/src/main/java/com/x8bit/bitwarden/ui/tools/feature/generator/GeneratorViewModel.kt

```diff
@@ -2622,6 +2629,13 @@ private fun UsernameGenerationOptions.ForwardedEmailServiceType?.toServiceType(
             )
         }
 
+        UsernameGenerationOptions.ForwardedEmailServiceType.FORWARD_EMAIL -> {
+            ForwardEmail(
```

Comment:
Should we add a test for this case?

Reply:
Yea. I thought my coverage report said it was hit. It must have been outdated, obviously. Thanks for catching this.

Reply:
None of this was conversion logic was being tested before, which is how this bug slipped by. I've refactored some things to get coverage and hopefully prevent that from happening again.